src/vm/eventpipe.cpp

```diff
@@ -716,11 +716,14 @@ bool EventPipe::WalkManagedStackForThread(Thread *pThread, StackContents &stackC
 
     stackContents.Reset();
 
+    bool gcOnTransitions = GC_ON_TRANSITIONS(FALSE);                // dont do GC for GCStress 3
```

Comment:
I would add a comment back to the issue that describes why we have disabled GCStress here.